packages/ndla-ui/src/LicenseByline/LicenseBylineDescriptionList.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from "@emotion/styled";
+import { spacing } from "@ndla/core";
+import { getLicenseRightByAbbreviation } from "@ndla/licenses";
+
+interface LicenseItemProps {
+  licenseRight: string;
+  locale?: string;
+}
+
+const StyledListItem = styled.li`
+  padding-bottom: ${spacing.xsmall};
+`;
+
+const LicenseItem = ({ licenseRight, locale }: LicenseItemProps) => {
+  const { description } = getLicenseRightByAbbreviation(licenseRight, locale);
+
+  return <StyledListItem>{description}</StyledListItem>;
+};
+
+interface LicenseDescriptionListProps {
+  licenseRights: string[];
+  locale?: string;
+}
+
+const StyledList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  list-style-type: disc;
+`;
+
+const LicenseBylineDescriptionList = ({ licenseRights, locale }: LicenseDescriptionListProps) => (
+  <StyledList>
+    {/* Filter away the CC-rights description since it isn't showed in the list anymore */}
+    {licenseRights
+      .filter((right) => right !== "cc")
+      .map((licenseRight) => (
+        <LicenseItem key={licenseRight} licenseRight={licenseRight} locale={locale} />
```

Comment:
Her kunne du sikkert inlina koden fra LicenseItem for å forenkle litt. LicenseItem gir ikkje veldig masse meirverdi.